src/GovUk.Education.ExploreEducationStatistics.Data.Api/GovUk.Education.ExploreEducationStatistics.Data.Api.csproj

```diff
@@ -13,7 +13,6 @@
     <NoWarn>$(NoWarn);1591</NoWarn>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="AspectInjector" Version="2.8.2" PrivateAssets="all" />
```

Comment:
I think we should be able to get rid of the following code block from `renovate.json` now:

```json
    {
    "description": "Disable AspectInjector and Aspects.Universal due to an issue with AspectInjector 2.9.0. See https://github.com/pamidur/aspect-injector/issues/244",
    "matchDatasources": ["nuget"],
    "matchPackageNames": ["AspectInjector", "Aspects.Universal"],
    "enabled": false
    }
```